prometheus/base/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- prometheus.rules.yaml
+- recording
+- alerting
```

Comment:
I don't think the directory structure here allows running `kustomize build`. Before we could `kustomize build prometheus/base`, but this does not work with the proposed structure as far as I can tell.

You'd need to add `kustomization.yaml` in the new directories you created and have each of them reference the yaml on that directory.